src/components/table/examples/table-selectable-row.tsx

```diff
@@ -0,0 +1,38 @@
+import { Component, h, State } from '@stencil/core';
+import { Column } from '../table.types';
+import { persons, columns, Person } from './persons';
+
+@Component({
+    tag: 'limel-example-table-selectable-row',
+    styleUrl: 'table.scss',
+    shadow: true,
+})
+export class TableExampleSelectableRow {
+    @State()
+    private selectedRaw: Person;
```

Comment:
R**a**w and on the lines 30 and 36 as well :)

Reply:
Haha

Reply:
Rawwww 🐯